staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -637,6 +637,9 @@ message WindowsContainerSecurityContext {
     // exist in the container image and be resolved there by the runtime;
     // otherwise, the runtime MUST return error.
     string run_as_username = 1;
+
+    // The contents of the GMSA credential spec to use to run this container.
+    string credential_spec = 2;
```

Comment:
The API change LGTM. We may need to change the comment to make it clear if we ever supports non-GMSA spects.

Reply:
@yujuhong : sorry if I'm misunderstanding, but that field's name was requested to not include "GMSA" in https://github.com/kubernetes/kubernetes/pull/75459#discussion_r283044104, and the comment makes it clear it's for GMSA for now; so not sure what else we should do here?

Reply:
I think this is fine for now. The field name matches the oci field name, and the description is reasonably consistent with the [oci doc](https://github.com/opencontainers/runtime-spec/blob/master/specs-go/config.go#L442-L443).